lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -143,6 +147,37 @@ for barclamp in $deactivation_list; do
 done
 
 
+### Stop pacemaker resources in all clusters and put the nodes into standby mode
+rm -f "${UPGRADE_DATA_DIR}/cluster_founders"
+
+# First check if cluster's health is good
+failed_actions_founder=
+for founder in $cluster_founders; do
+  if ssh ${founder} 'LANG=C crm status | grep -q "^Failed actions:"'; then
+    failed_actions_founder=$founder
+  fi
+done
+if [ -n "$failed_actions_founder" ] ; then
+  die "'crm status' command reports some failed actions at $failed_actions_founder node.\nClean pacemaker resources using 'crm resource cleanup' before proceeding with the upgrade."
+fi
```

Comment:
This check should be done before we deactivate barclamps, imho (around https://github.com/SUSE-Cloud/suse-cloud-upgrade/pull/29/files#diff-bea8a81bfd95a7d3c9f6362b7d71b76dR118 -- btw, we can remove that old check there)